src/main/java/com/boardgo/common/utils/CookieUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.boardgo.common.utils;
+
+import static com.boardgo.common.constant.TimeConstant.ACCESS_TOKEN_DURATION;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.Objects;
+import java.util.Optional;
+
+public abstract class CookieUtil {
+
+    public static Optional<Cookie> getCookie(HttpServletRequest request, String name) {
+        Cookie[] cookies = request.getCookies();
+
+        if (!Objects.isNull(cookies)) {
+            for (Cookie cookie : cookies) {
+                if (name.equals(cookie.getName())) {
+                    return Optional.of(cookie);
+                }
+            }
+        }
```

Comment:
```suggestion
        if (Objects.isNull(cookies)) {
            return Optional.empty();
        }    
        for (Cookie cookie : cookies) {
                if (name.equals(cookie.getName())) {
                    return Optional.of(cookie);
                }
            }
```

이게 조금 더 좋아보입니다! (조심스레..)

Reply:
여기서 Optional로 cookie를 내보내는 이유가 궁금합니다!

Reply:
Cookie 를 요청한 곳에서 반환된 쿠키가 없을 경우 예외를 발생하게 하기 위해서 Optional 처리했어요!
제안주신 코드로 수정할게요!